dockerfiles/vcpkg/ubuntu-22.04.dockerfile

```diff
@@ -1,6 +1,5 @@
 ARG VCPKG_REF=latest
-ARG DOCKER_NAMESPACE=hpccbuilds
-FROM ${DOCKER_NAMESPACE}/vcpkg-ubuntu-22.04:${VCPKG_REF}
+FROM hpccsystems/platform-build-base-ubuntu-22.04:$VCPKG_REF
```

Comment:
question: is "hpccbuilds" no longer used, everything to/from "hpccsystems" ?

Reply:
It is for some of the caches